sysrib/server.go

```diff
@@ -521,15 +589,18 @@ func convertZebraRoute(niName string, zroute *zebra.IPRouteBody) *Route {
 			NetworkInstance: niName,
 		})
 	}
+	var routePref RoutePreference
+	switch zroute.Type {
+	case zebra.RouteBGP:
+		routePref.AdminDistance = 20
```

Comment:
should this be configureable? if so TODO, if not make it const

Reply:
Made const.